apps/docs/azure-functions/src/lib/auth.ts

```
@@ -2,50 +2,130 @@
+      operation: "validateToken",
+    });
+    return null;
+  }
+}
+
 /**
  * Extract user ID from request (from validated token)
- * In production, validate the JWT token properly
  */
-export function getUserIdFromRequest(request: HttpRequest): string | null {
+export async function getUserIdFromRequest(
```

ESLint

[error] Multiple exports of name 'getUserIdFromRequest'.

(import/export)